pkg/util/detector/detector.go

```diff
@@ -199,6 +200,11 @@ func (d *ResourceDetector) Reconcile(key util.QueueKey) error {
 // All objects which API group defined by Karmada should be ignored:
 // - cluster.karmada.io
 // - policy.karmada.io
+//
+//
+// if skippedPropagatingAPIs is set, Karmada will not ignore these apis.
```

Comment:
The api objects listed above need to be ignored anyway (or leasted by default), we don't want users to manually input the things they don't care when trying to skip something else.